src/redux/migrations.test.ts

```diff
@@ -283,7 +282,13 @@ describe('Redux persist migrations', () => {
     const migratedSchema = migrations[12](stub)
     expect(migratedSchema.app).toEqual(appStub)
     expect(migratedSchema.exchange.otherExchangeProps).toEqual(exchangeStub)
-    expect(migratedSchema.exchange.history).toEqual(exchangeInitialState.history)
```

Comment:
Why does this old migration test need to change?

Reply:
this is changed because `exchangeInitialState` no longer exists because it was imported from a file that is now deleted. I'll change it to importing from migrations.ts